src/controllers/ProjectController.ts

```diff
@@ -0,0 +1,16 @@
+import express from "express";
```

Comment:
Can import like `import { Request, Response } from "express";`

So on likes 4 it can be 
`export const getProjectForm = async (req: Request, res: Response): Promise<void> => {`